packages/components/src/custom-select-control/test/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
+/**
+ * WordPress dependencies
+ */
+import { CustomSelectControl } from '@wordpress/components';
+
+describe( 'CustomSelectControl', () => {
+	const onKeyDown = jest.fn();
+
+	it( 'Captures the keypress event and does not let it propagate', () => {
+		const wrapper = (
+			<div
+				role="none"
+				className="test-class-name"
+				onKeyDown={ onKeyDown }
+			>
+				<CustomSelectControl options={ [ 'a', 'b', 'c' ] } />
+			</div>
+		);
+		const { container } = render( wrapper );
```

Comment:
We could simplify the code by rendering the markup directly inside the `render` function:

```js
const { container } = render(
  <div role="none" onKeyDown={ onKeyDown }>
    <CustomSelectControl options={ options } />
  </div>
);
```

Actually, since we don't need `container` (see next comment), this could be just written as

```js
render(
  <div role="none" onKeyDown={ onKeyDown }>
    <CustomSelectControl options={ options } />
  </div>
);
```

Reply:
Thanks, this is great to know :)